composer.json

```diff
@@ -7,6 +7,7 @@
     }
   },
   "require": {
+    "php": ">=5.6.0",
```

Comment:
I think it would be wiser to support a lower version of PHP like 5.3, as many deployment are made on CentOS and I dont think they are up to latest PHP